devito/symbolics/manipulation.py

```diff
@@ -13,8 +13,8 @@
 
 def freeze(expr):
     """
-    Reconstruct ``expr`` turning all :class:`sympy.Mul` and :class:`sympy.Add`
-    into :class:`FrozenExpr` equivalents.
+    Reconstruct ``expr`` turning all sympy.Mul` and sympy.Add
```

Comment:
your forgot a `

Reply:
oh I have fixed that and forgot to push.